creator-node/src/components/tracks/trackContentUploadManager.js

```diff
@@ -0,0 +1,326 @@
+const path = require('path')
```

Comment:
pretty much just moving /track_content_async logic into a class called TrackContentUploadManager. slight changes in structure, like in `transcodeAndSegment()`